hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/OMTrashRecoverResponse.java

```diff
@@ -55,8 +55,10 @@ public void addToDBBatch(OMMetadataManager omMetadataManager,
     String trashKey = omMetadataManager
         .getOzoneKey(omKeyInfo.getVolumeName(),
             omKeyInfo.getBucketName(), omKeyInfo.getKeyName());
+    String deleteKey = omMetadataManager.getOzoneDeletePathKey(
+        omKeyInfo.getObjectID(), trashKey);
     RepeatedOmKeyInfo repeatedOmKeyInfo = omMetadataManager
-        .getDeletedTable().get(trashKey);
+        .getDeletedTable().get(deleteKey);
```

Comment:
@sumitagrawl , It seems that `TrashRecover` won't work if there are old keys (without object id) in the deletedTable?

Reply:
@szetszwo This code is unused and needs to be removed, a JIRA [HDDS-8531](https://issues.apache.org/jira/browse/HDDS-8531) is created by @sadanand48 for handling same.

This does not have any problem.

Reply:
If the code is unused, why changing it?